ui/k-info-panel/switch_info.kytos

```diff
@@ -61,7 +66,36 @@
        window.d3.json(endpoint, function(error, result) {
          self.flows = result[dpid].flows
        })
-     }
+     },
+     get_enabled_value() {
+       this.next_state = this.metadata.enabled == 'true'? 'Disable' : 'Enable'
+     },
+     msg_state_success(data){
+       let notification = {
+         title: 'Switch ' + this.next_state + 'd: Succeed',
+         description: 'The switch ' + this.metadata.dpid + ' was ' + this.next_state.toLowerCase() + 'd.',
+         icon: 'gear',
+       }
+       this.next_state = this.next_state == 'Enable'? 'Disable' : 'Enable'
```

Comment:
@Alopalao, let's also make sure to update the `this.metadata.enabled` attribute acordingly as you've done for `this.next_state`, otherwise the displayed data on basic details would be inconsistent since they depend on the props to be updated again. Currently, if you click in the the button the `metadata.enabled` value would still display the old value.

Reply:
Now the metadata detail updates after changing the state of `enabled`. Commit [775ac47dc6d2dbb931bcdc304c6d67f9c3ab8242](https://github.com/kytos-ng/topology/pull/70/commits/775ac47dc6d2dbb931bcdc304c6d67f9c3ab8242)